src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -536,7 +544,7 @@ int32_t SystemNative_SetAddressFamily(uint8_t* socketAddress, int32_t socketAddr
 {
     struct sockaddr* sockAddr = (struct sockaddr*)socketAddress;
     if (sockAddr == NULL || socketAddressLen < 0 ||
-        !IsInBounds((const uint8_t*)sockAddr, (size_t)socketAddressLen, (const uint8_t*)&sockAddr->sa_family, sizeof_member(struct sockaddr, sa_family)))
+        !IsInBounds((const uint8_t*)sockAddr, (size_t)socketAddressLen, (const uint8_t*)&sockAddr->sa_family, sizeof_member(sockaddr, sa_family)))
```

Comment:
Suggest IsInBounds take a const void* and then remove all the casts on inputs to it.

Reply:
Yes, that would be nice.

Reply:
Will do separately later thank you.